packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -105,6 +107,27 @@ export const useSchedule = ({
       (Boolean(eventSlug) || Boolean(eventId) || eventId === 0),
   };
 
+  const teamScheduleFromApiV2 = useApiV2AvailableSlots({
+    ...input,
+    duration: input.duration ? Number(input.duration) : undefined,
+    routedTeamMemberIds: input.routedTeamMemberIds ?? undefined,
+    teamMemberEmail: input.teamMemberEmail ?? undefined,
+    eventTypeId: eventId ?? undefined,
+    enabled: Boolean(isTeamEvent),
+  });
+
+  if (isTeamEvent && teamScheduleFromApiV2.isSuccess && teamScheduleFromApiV2.data) {
+    return {
```

Comment:
i just made a refactor so that we fall back to trpc query if apiv2 query fails for some reason